### ISSUE TITLE
カスタムスケジューラを登録したい／I want to register a custom scheduler

### ISSUE THREAD

**Issue by muooon** [kohya-ss/sd-scripts#2365]
こんにちは。表題のとおりなんですが、、、
カスタムスケジューラをオプティマイザのように簡単に登録できませんか？

---

Hello. As the subject line suggests...
Is there a way to register a custom scheduler as easily as an optimizer ?

---

https://github.com/muooon/EmoSens/tree/v3.9.0_ecc/scheduler

**Comment by kohya-ss** [comment]
`--lr_scheduler_type`と`--lr_scheduler_args`がOptimizerと同様に使えると思います。もし何か問題ありましたらお知らせください。

`--lr_scheduler_type` and `--lr_scheduler_args` will work for a scheduler like for an optimizer. Please let us know if you encounter any problems.

**Comment by muooon** [comment]
ありがとうございました。スケジューラ側の不備を修正し無事につかうことができました。
SD-Script は、本当にすごいです！ ここまで柔軟な設計で動かせる、驚きと感動でいっぱいです。
感謝を込めて Close, 解決済みとさせてください。本当にありがとうございました。

Thank you very much. I fixed the issue on the scheduler side and was able to use it without any problems.
SD-Script is truly amazing! I’m filled with surprise and admiration that it can operate with such a flexible design.
With my sincere gratitude, I’d like to mark this as “Closed” and “Resolved.” Thank you so much.
